tests/setup/mod.rs

```diff
@@ -183,6 +186,12 @@ impl Env {
             .user_account
             .create_transaction(a(BURROWLAND_ID))
             .deploy_contract(BURROWLAND_WASM_BYTES.to_vec())
+            .function_call(
+                "migrate_state".to_string(),
+                b"{}".to_vec(),
+                DEFAULT_GAS.0,
+                0,
+            )
```

Comment:
Example how the state migration is done. It's in the same transaction as a contract bytes upgrade. If one of the actions fails, then the entire receipt reverted, so the upgrade wouldn't happen.

Once the remote upgrade is implemented, it will be also done in one receipt, but the 2 actions receipt will be originated by the internal method instead of the external transaction by the contract account.